source/Coop.Core/Server/Services/Settlements/Messages/SettlementOwnershipChangeApproved.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Common.Messaging;
+using ProtoBuf;
+
+namespace Coop.Core.Server.Services.Settlements.Messages
+{
+    /// <summary>
+    /// Event for settlement ownership change approved sent from server to client
+    /// </summary>
+    [ProtoContract(SkipConstructor = true)]
+    public class SettlementOwnershipChangeApproved : ICommand
```

Comment:
Please create an integration test for this message. See https://github.com/Bannerlord-Coop-Team/BannerlordCoop/blob/development/source/Coop.IntegrationTests/MobileParties/PartyBehaviorTest.cs